proto/pulumi/resource.proto

```diff
@@ -243,6 +244,24 @@ message TransformResponse {
     TransformResourceOptions options = 2; // the options for the resource.
 }
 
+message TransformInvokeRequest {
```

Comment:
I know we've not been very good at it in this file/historically, but I think we should doc-comment the types too.